### PR TITLE
remove duplicate keys from istiooperator crd in docs and setup scripts

### PR DESCRIPTION
**Pull request by joekelley**
BOT NOTES: 
resolves https://github.com/solo-io/gloo-mesh/issues/1251

### CODE REVIEW

File: docs/content/guides/installing_istio.md [hunk 1]
```diff
@@ -114,8 +114,9 @@ metadata:
   name: example-istiooperator
   namespace: istio-system
 spec:
-  profile: minimal
+  profile: preview
```
Comment (jenshu) [inline docs/content/guides/installing_istio.md:117]: what effect will this profile change have?

Comment (joekelley) [reply docs/content/guides/installing_istio.md:117]: need to confirm but i believe this is required to enable Istio Smart DNS

Comment (joekelley) [reply docs/content/guides/installing_istio.md:117]: not needed -- thanks @Sodman !

**Review by Sodman — APPROVED** [review]
👍

**Pull request merged**